src/extension/serializer.ts

```diff
@@ -769,16 +786,10 @@ export class GrpcSerializer extends SerializerBase {
 
     return result
   }
-
-  static sessionOutputsEnabled() {
-    const isAutoSaveOn = ContextState.getKey<boolean>(NOTEBOOK_AUTOSAVE_ON)
-    const isSessionOutputs = getSessionOutputs()
-
-    return isSessionOutputs && isAutoSaveOn
-  }
-
+  // await (ser as any).cacheNotebookOutputs(fixture, 'irrelevant') ???
   private async cacheNotebookOutputs(
-    notebook: Notebook,
+    notebook: typeof this.protoNotebookType,
+    maskedNotebook: typeof this.protoNotebookType,
```

Comment:
Similar to question above; why did this method change to now take the masked notebook?

Reply:
see above